platform-core-module/src/main/java/com/softicar/platform/core/module/program/execution/ProgramExecutionRunnable.java

```diff
@@ -95,5 +111,21 @@ private void updateOutputAndTerminatedAt() {
 				.save();
 			transaction.commit();
 		}
+		logFilePath.toFile().delete();
+	}
+
+	private Path getLogFilePath(AGProgramExecution execution) {
+
+		String logFilename = "softicar-program-execution-%s.log".formatted(execution.getId());
+		return getLogDirectory().resolve(logFilename);
+	}
+
+	private Path getLogDirectory() {
+
+		if (AGCoreModuleInstance.getInstance().isTestSystem()) {
+			return Path.of(SystemPropertyEnum.JAVA_IO_TMPDIR.get());
+		} else {
+			return SofticarHome.getLogPath();
+		}
```

Comment:
- IMO, I think it would be better to add a new field `AGCoreModuleInstance.PROGRAM_LOG_FOLDER` with default `""`
  - rationale: no hard-coding of additional paths
- If that field is blank, `SystemPropertyEnum.JAVA_IO_TMPDIR.get()` is used.

Reply:
If the systems are configured the same, or arbitrarily different is not the decision of the developer. IMO.

Reply:
To be clear, I will not disapprove this PR because of this, but I highly recommend to make things configurable via DB and not hard-code it in the source code. When was hard-coding any externals a good idea? ;)